client/src/main/java/io/split/client/LocalhostSplitClient.java

```diff
@@ -18,13 +19,13 @@
  * @author adil
  */
 public final class LocalhostSplitClient implements SplitClient {
-    private LocalhostSplitFactory _container;
-    private ImmutableMap<String, String> _featureToTreatmentMap;
+    private static final Logger _log = LoggerFactory.getLogger(LocalhostSplitClient.class);
 
-    public LocalhostSplitClient(LocalhostSplitFactory container, Map<String, String> featureToTreatmentMap) {
-        checkNotNull(featureToTreatmentMap, "featureToTreatmentMap must not be null");
-        _featureToTreatmentMap = ImmutableMap.copyOf(featureToTreatmentMap);
-        _container = container;
+    private Map<SplitAndKey, String> _map;
```

Comment:
can we name this anything other than map? for example: `_evaluationsOverride` ?

Reply:
it is not the overrides. It is the entire mapping read from .splits file